resources.auto.tfvars

```diff
@@ -0,0 +1,7 @@
+account_id              = "xxxxxxxxxxxx"
+image_id                = "ami-0dd0ab23" #Public Centos8 #eu-west-2
```

Comment:
Warning: image is [deprecated](https://docs.outscale.com/fr/userguide/R%C3%A9f%C3%A9rence-des-OMI-officielles.html) but still available. Can you use a public copy of the latest image please instead ? This will avoid future availability problems.

Reply:
done